File: go/array-consecutive/array-consecutive.go
package main

import(
	"fmt"
	"slices"
)

func makeArrayConsecutive2(statues []int)int{
	max := slices.Max(statues)
	min := slices.Min(statues)
	diff := max - min
	return diff - len(statues) + 1
}

func main() {
	statues := []int{6 , 2 , 3 , 8}
	fmt.Printf("The answer for the array %v is %d", statues, makeArrayConsecutive2(statues))
}

File: js/array-consecutive.js
function makeArrayConsecutive2(statues){
    let maxArray = Math.max(...statues)
    let minArray = Math.min(...statues)
    let diffArray = maxArray-minArray
    let result = (diffArray - statues.length) + 1
    return result

}

let statues = [6 , 2 , 3 , 8]
console.log(makeArrayConsecutive2(statues))
